connector/src/chain_data.rs

```diff
@@ -229,7 +237,7 @@ impl ChainData {
             .ok_or_else(|| anyhow::anyhow!("account {} has no live data", pubkey))
     }
 
-    pub fn iter_accounts<'a>(&'a self) -> impl Iterator<Item = (&'a Pubkey, &'a AccountData)> {
```

Comment:
why did you remove this? May be some other libraries are using these lifetimes.

Reply:
strange. I did not remove it - just removed the lifetime annotations.

this is the actual patch:

```
-    pub fn iter_accounts<'a>(&'a self) -> impl Iterator<Item = (&'a Pubkey, &'a AccountData)> {
+    pub fn iter_accounts(&self) -> impl Iterator<Item = (&Pubkey, &AccountData)> {
```